lib/screens/home/local_widget/province_card.dart

```diff
@@ -0,0 +1,238 @@
+                      },
+                    )
+                  : ListTile(
+                      title: Text('ស្រុក' + (district[index].khmer ?? '')),
+                      subtitle: Text('$commune និង $villageភូមិ'),
+                      onTap: () {
+                        Navigator.restorablePushNamed(context, RouteConfig.DISTRICT);
+                      },
+                    ),
+            ],
+          );
+        },
+      ),
+    );
+  }
+
+  ListTile buildTourPlaceListTile({
+    required BuildContext context,
+    String? title,
+    String? subtitle,
+    void Function()? onTap,
+  }) {
+    return ListTile(
+      title: Text(
+        title ?? '',
+        style: Theme.of(context)
+            .textTheme
+            .headline6
+            ?.copyWith(fontWeight: FontWeight.bold, color: Theme.of(context).primaryColor),
+      ),
+      subtitle: subtitle != null
+          ? Text(
+              subtitle,
+              style: Theme.of(context).textTheme.caption,
+            )
+          : null,
+      trailing: Icon(
+        Icons.keyboard_arrow_right,
+        color: Colors.black,
+      ),
+      onTap: onTap,
+    );
+  }
+
+  ListView buildInfoCount(BuildContext context) {
+    List<String> titles = [];
+    List<String> subtitles = [];
+    if (province.srok != 0) {
+      titles.add('ស្រុក');
+      subtitles.add(NumberHelper.toKhmer(province.srok));
+    }
+    if (province.khan != 0) {
+      titles.add('ខណ្ឌ');
+      subtitles.add(NumberHelper.toKhmer(province.khan));
+    }
+    if (province.sangkat != 0) {
+      titles.add('សង្កាត់');
+      subtitles.add(NumberHelper.toKhmer(province.sangkat));
+    }
+    if (province.commune != 0) {
+      titles.add('ឃុំ');
+      subtitles.add(NumberHelper.toKhmer(province.commune));
+    }
+    if (province.village != 0) {
+      titles.add('ភូមិ');
+      subtitles.add(NumberHelper.toKhmer(province.village));
+    }
+    return ListView(
+      scrollDirection: Axis.horizontal,
+      children: List.generate(
+        titles.length,
+        (index) {
+          return Container(
+            width: 97.5,
+            child: Column(
+              children: [
+                Text(
+                  titles[index],
+                  style: Theme.of(context).textTheme.bodyText1?.copyWith(color: Theme.of(context).primaryColor),
+                ),
+                Text('${subtitles[index]}'),
+              ],
+            ),
+          );
+        },
+      ),
+    );
+  }
+
+  Container buildProvinceHeader(BuildContext context) {
+    List<TbDistrictModel> krongs = [];
+    for (var dist in district) {
+      if (dist.type == "KRONG") {
+        krongs.add(dist);
+      }
+    }
+    String? krongTitle;
+    if (krongs.length == 0)
+      krongTitle = null;
+    else {
+      krongTitle =
+          krongs.length == 2 ? 'ក្រុង${krongs[0].khmer} និង ក្រុង${krongs[1].khmer}' : 'ក្រុង${krongs[0].khmer}';
```

Comment:
good solution since we only have 2 max cities but you can also use join instead: 
`krongTitle = "ក្រុង" + krongs.join(" និង ក្រុង")`.